docs/developers_guide/api.md

```diff
@@ -0,0 +1,65 @@
+(dev-api)=
+
+# API reference
+
+This page provides an auto-generated summary of the mache API. For more
+details and examples, refer to the relevant sections in the main part of the
+documentation.
+
+## discover
+
+```{eval-rst}
+.. currentmodule:: mache.discover
+
+.. autosummary::
+    :toctree: generated/
```

Comment:
Here and elsewhere, everything after `autosummary` needs to be indented 3 spaces, not 4.  I don't know why sphinx is so picky about that but it is...

```suggestion
   :toctree: generated/
```